core/controllers/feedback_test.py

```diff
@@ -643,7 +643,6 @@ def setUp(self):
         self.can_send_feedback_email_ctx = self.swap(
             feconf, 'CAN_SEND_FEEDBACK_MESSAGE_EMAILS', True)
 
-
     def test_that_emails_are_sent(self):
         with self.can_send_emails_ctx, self.can_send_feedback_email_ctx:
```

Comment:
yeah added test at controller level to check that email body is correct so in case of single or multiple feedback message.